DockerFile

```diff
@@ -0,0 +1,26 @@
+ARG PYTHON_VERSION=3.10-slim-buster
+
+FROM python:${PYTHON_VERSION}
+
+ENV PYTHONDONTWRITEBYTECODE 1
+ENV PYTHONUNBUFFERED 1
+
+RUN mkdir -p /code
+
+WORKDIR /code
+
+COPY requirements.txt /tmp/requirements.txt
+
+RUN set -ex && \
+    pip install --upgrade pip && \
+    pip install -r /tmp/requirements.txt && \
+    rm -rf /root/.cache/
+
+COPY . /code/
+
+# RUN python manage.py collectstatic --noinput
+
+EXPOSE 8000
+
+# replace demo.wsgi with <project_name>.wsgi
+CMD ["gunicorn", "--bind", ":8000", "--workers", "2", "django_app.wsgi"]
```

Comment:
Whitenoise is used in this project intead of gunicorn, there is no need to include gunicorn conf

Reply:
I used whitenoise to serve static files not sure if it can replace gunicorn.. i will check